src/main/java/io/cryostat/net/web/http/api/beta/RecordingsFromIdPostHandler.java

```diff
@@ -172,21 +174,27 @@ public List<HttpMimeType> consumes() {
     }
 
     @Override
-    public void handleAuthenticated(RoutingContext ctx) throws Exception {
+    public SecurityContext securityContext(RoutingContext ctx) {
+        return auth.contextFor(
+                idHelper.reverseLookup(ctx.pathParam("jvmId"))
+                        .orElseThrow(() -> new HttpException(404)));
+    }
 
+    @Override
+    public void handleAuthenticated(RoutingContext ctx) throws Exception {
         if (!fs.isDirectory(savedRecordingsPath)) {
-            throw new ApiException(503, "Recording saving not available");
+            throw new HttpException(503, "Recording saving not available");
```

Comment:
What's the reason for replace ApiException with HttpException?

Reply:
Probably another rebase mixup or something.

Reply:
I don't remember now... it does extend the base class `v1` `AbstractAuthenticatedRequestHandler`, not the `v2+` `AbstractV2RequestHandler`, but `AbstractAuthenticatedRequestHandler` does know to treat them the same and just rethrow them for the `WebServer` to handle, so I'm not sure what functional difference this would make if any. Conceptually it should be the same thing.